uint16 current
uint16 pressure
uint16 angle
uint16 count
uint8  mask
